Milestone4

jofer86 opened the pull request:
The first implementation of milestone 4 for the React Calculator project. Waiting on feedback as some changes might be needed. :+1:

SunnyAsar commented on src/logic/operate.js:
```diff
@@ -0,0 +1,20 @@
+    '-': 'minus',
+    '÷': 'div',
+    x: 'times',
+  };
```
Hey, we are missing the calculations here. use `switch case` on the operator and when the case is `+` return num1 + num2 for example and so on.

jofer86 replied:
- @SunnyAsar the calculation is made in the return statement, thats how the Big.js library does the calculation. num1[oper](num2) is the same as passing num1.sum(num2) etc. So switch statements are not necesary as the calculation is made using object bracket notation to pass the appropiate operator required by the library.

jofer86 commented:
Fixed divide by zero issues, moved the style folder away from the component folder. Just left the index.css file in the src folder, as I think it might break some stuff so I just left it there. I also fixed the linting issues. All checks are passing now.
